instance/models/instance.py

```diff
@@ -440,6 +470,7 @@ def provision(self):
                 self.logger.info('Provisioning external databases...')
                 self.provision_mysql()
                 self.provision_mongo()
+                self.provision_swift()
```

Comment:
nit: it would be nice to put a log statement before each step: "Provisioning MySQL...", "Provisioning Mongo...", "Provisioning Swift...".